chart/kubeapps/values.yaml

```diff
@@ -1574,27 +1574,30 @@ pinnipedProxy:
     ##   - myRegistryKeySecretName
     ##
     pullSecrets: []
-  ## @param pinnipedProxy.defaultPinnipedNamespace Specify the (default) namespace in which pinniped concierge is installed
+  ## @param pinnipedProxy.defaultPinnipedNamespace Namespace in which pinniped concierge is installed
   ##
   defaultPinnipedNamespace: pinniped-concierge
-  ## @param pinnipedProxy.defaultAuthenticatorType Specify the (default) authenticator type
+  ## @param pinnipedProxy.defaultAuthenticatorType Authenticator type
   ##
   defaultAuthenticatorType: JWTAuthenticator
-  ## @param pinnipedProxy.defaultAuthenticatorName Specify the (default) authenticator name
+  ## @param pinnipedProxy.defaultAuthenticatorName Authenticator name
   ##
   defaultAuthenticatorName: jwt-authenticator
-  ## @param pinnipedProxy.defaultPinnipedAPISuffix Specify the (default) API suffix
+  ## @param pinnipedProxy.defaultPinnipedAPISuffix API suffix
   ##
   defaultPinnipedAPISuffix: pinniped.dev
-  ## @param pinnipedProxy.TLSSecret Specify an optional TLS secret with which to proxy requests
+  ## TLS settings for Pinniped Proxy
+  ## @param pinnipedProxy.tls.existingSecret TLS secret with which to proxy requests
+  ## @param pinnipedProxy.tls.caCertificate TLS CA cert config map which clients of pinniped proxy should use with TLS requests
+  ## This config map must contain a ca.crt key with the CA cert content as the value.
   ##
-  TLSSecret: ""
-  ## @param pinnipedProxy.CACert Specify the TLS CA cert config map which
-  ## clients of pinniped proxy should use with tls requests. This config map
-  ## must contain a ca.crt key with the CA cert content as the value.
-  ##
-  CACert: ""
-  ## @param pinnipedProxy.lifecycleHooks for the Pinniped Proxy container(s) to automate configuration before or after startup
+  tls:
+    ## Optional TLS secret with which to proxy requests
+    existingSecret: ""
+    ## TLS CA cert config map which clients of pinniped proxy should use with tls requests.
+    ## This config map must contain a ca.crt key with the CA cert content as the value.
+    caCertificate: ""
+  ## @param pinnipedProxy.lifecycleHooks For the Pinniped Proxy container(s) to automate configuration before or after startup
   ##
```

Comment:
What about this format? This way we avoid repeating the explanation, no? 

```yaml
...
  defaultPinnipedAPISuffix: pinniped.dev
  ## TLS settings for Pinniped Proxy
  ## ref: https://kubeapps.dev/docs/latest/howto/oidc/using-an-oidc-provider-with-pinniped/#running-the-pinniped-proxy-service-over-tls
  tls:
    ## @param pinnipedProxy.tls.existingSecret TLS secret with which to proxy requests
    ##
    existingSecret: ""
    ## @param pinnipedProxy.tls.caCertificate TLS CA cert config map which clients of pinniped proxy should use with TLS requests
    ## This config map must contain a ca.crt key with the CA cert content as the value.
    ##
    caCertificate: ""
  ## @param pinnipedProxy.lifecycleHooks For the Pinniped Proxy container(s) to automate configuration before or after startup
  ##
  lifecycleHooks: {}
...
```

Reply:
Ok, done. I thought of doing this way, but saw the blocks above in the `values.yaml` and all params where aggregated in the parent one. Didn't know it could be done in this way.

Reply:
I'd recall the support for these non-aggregated parms was added after we built the values.yaml, but now that it's supported, we better use it. In fact, I've double-checked it running the readmenator manually because I wasn'ts sure haha